src/quequations.jl

```diff
@@ -74,10 +86,14 @@ end
 @doc """
 Lindblad operator construct from the `Hamiltonian` and `collapse operators`
 
-Input Parameters :
+### Arguments
+
+Inputs :
+* hamiltonian   : hamiltonian of the system
+* collapse_ops  : Collapse operators
 
-`hamiltonian` : Hamiltonain of the system
-`collapse_ops` : Collapse operators
+Output :
+* Lindblad operator construct with structure same as hamiltonian.
```

Comment:
This I don't understand. The Lindblad operator/matrix is constructed from the Hamiltonian and the collapse operators.